scripts/setup_osx.sh

```diff
@@ -1,38 +1,265 @@
+    "correctly under your current OS X version (${VER}). If the installation" \
+    "is successful, please update the list of valid versions in this script." \
+    "Otherwise, if you are able to fix your compatibility issue, please file" \
+    "an issue to (https://github.com/rlworkgroup/garage/issues) and include a"\
+    "link to your Pull Request.\n" |
+  fold -s
+  while [[ "${continue_var}" != "y" ]]; do
+    read -p "Continue? (y/n): " continue_var
+    if [[ "${continue_var}" = "n" ]]; then
+      exit
+    fi
+  done
 fi
 
-echo "Creating conda environment..."
+# Verify there's a file in the mjkey path
+test -f "${_arg_mjkey_path}" || _PRINT_HELP=yes die \
+  "The path ${_arg_mjkey_path} of the MuJoCo key is not valid." 1
+
+# Make sure that we're under the garage directory
+GARAGE_DIR="$(dirname $(script_dir_path))"
+cd "${GARAGE_DIR}"
+
+# File where environment variables are stored
+BASH_PROF="${HOME}/.bash_profile"
+
+# Install dependencies
+echo "Installing garage dependencies"
+echo "You will probably be asked for your sudo password"
+
+# Homebrew is required first to install the other dependencies
+hash brew 2>/dev/null || {
+  # Install the Xcode Command Line Tools
+  xcode-select --install
+  # Install Homebrew
+  /usr/bin/ruby -e "$(curl -fsSL \
+    https://raw.githubusercontent.com/Homebrew/install/master/install)"
+}
+
+# For installing garage: bzip2, git, glfw, unzip, wget
+# For building glfw: cmake
+# Required for OpenAI gym: cmake boost boost-python ffmpeg sdl2 swig wget
+# Required for OpenAI baselines: cmake openmpi
+brew update
+brew install \
+  bzip2 \
+  git \
+  glfw \
+  unzip \
+  wget \
+  cmake \
+  boost \
+  boost-python \
+  ffmpeg \
+  sdl2 \
+  swig \
+  openmpi
+
+# Leave a note in ~/.bashrc for the added environment variables
+if [[ "${_arg_set_envvar}" = on ]]; then
+  echo -e "\n# Added by the garage installer" >> "${BASH_PROF}"
+fi
+
+# Set up MuJoCo
+if [[ ! -d "${HOME}"/.mujoco/mjpro150 ]]; then
+  mkdir "${HOME}"/.mujoco
+  MUJOCO_ZIP="$(mktemp -d)/mujoco.zip"
+  wget https://www.roboti.us/download/mjpro150_osx.zip -O "${MUJOCO_ZIP}"
+  unzip -u "${MUJOCO_ZIP}" -d "${HOME}"/.mujoco
+else
+  print_warning "MuJoCo is already installed"
+fi
+# Configure MuJoCo as a shared library
+export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:${HOME}/.mujoco/mjpro150/bin"
+LD_LIB_ENV_VAR="LD_LIBRARY_PATH=\"\$LD_LIBRARY_PATH:${HOME}/.mujoco/mjpro150"
+LD_LIB_ENV_VAR="${LD_LIB_ENV_VAR}/bin\""
+if [[ "${_arg_set_envvar}" = on ]]; then
+  echo "export ${LD_LIB_ENV_VAR}" >> "${BASH_PROF}"
+fi
+
+# Set up conda
+hash conda 2>/dev/null || {
+  CONDA_INSTALLER="$(mktemp -d)/miniconda.sh"
+  sudo chmod u+x "${CONDA_INSTALLER}"
+  wget https://repo.continuum.io/miniconda/Miniconda2-latest-MacOSX-x86_64.sh \
+    -O "${CONDA_INSTALLER}"
+  sudo bash "${CONDA_INSTALLER}" -b
+  # Add conda to executable programs
+  CONDA_PATH="${HOME}/miniconda2/bin/"
+  export PATH="$PATH:${CONDA_PATH}"
```

Comment:
conda doesn't use PATH anymore. instead you execute a script within your bashrc/bash_profile

Reply:
(see the screenshot above)